packages/remix-server-runtime/magicExports/remix.ts

```diff
@@ -34,13 +34,9 @@ export type {
 } from "@remix-run/server-runtime";
 
 export {
-  createCookie,
   isCookie,
   createSession,
   isSession,
-  createSessionStorage,
-  createCookieSessionStorage,
-  createMemorySessionStorage,
```

Comment:
TODO: for backwards compatibility, we need to add `createCookie`, `createSessionStorate`, `createCookieSessionStorage`, and `createMemorySessionStorage` to the magic exports for each server runtime package.